### ISSUE TITLE
INTERNAL SERVER ERROR: api.glygen.org search result list download...

### ISSUE THREAD

**Issue by edwardsnj** [glygener/glygen-issues#583]
https://api.glygen.org/glycan/search?query=%7B%22glycan_type%22%3A+%22N-linked%22%7D
returns: {"list_id": "ac10fda4d13ee25906ba343750cab26c"}

POST to https://api.glygen.org/data/list_download
with payload: {"compressed": false, "type": "glycan_list", "id": "ac10fda4d13ee25906ba343750cab26c", "format": "csv"}

This returns HTTP Error 500: INTERNAL SERVER ERROR

Please look at this ASAP, this is blocking my GNOme update, which is part of the Glycan Data build.

**Comment by rykahsay** [comment]
It was slow but I was able to download it (screenshot given below). 

<img width="1426" alt="image" src="https://github.com/glygener/glygen-issues/assets/20584876/85b074a2-8517-44a2-9cc5-586768283772">

**Comment by edwardsnj** [comment]
I am not using the website, I am using the API.

Please check the API.

**Comment by rykahsay** [comment]
Sorry, looking at it now

**Comment by rykahsay** [comment]
I think I found the problem. It expects "download_type" instead of "type".  I have updated the API documentation now.

<img width="619" alt="image" src="https://github.com/glygener/glygen-issues/assets/20584876/d05e7407-3775-4fb8-9912-769732882484">

**Comment by edwardsnj** [comment]
Great, that fixed the problem. So you will update the documentation, rather than the service? Just checking.

**Comment by rykahsay** [comment]
When we released 2.1, that parameter was changed from "type" to "download_type" and was communicated to Sujeet (that's why the frontend was working), but I didn't update the swagger documentation (it was still asking users to provide "type" instead of "download_type"). But now, I have updated it and it asks the users to provide "download_type".

Whenever an API changes in terms of expected arguments, my "minimal responsibility" would be to update the swagger documentation, which I failed do in this case.
